### PR TITLE
Simplification Total RAM getter

### PR DESCRIPTION
**Pull request by panther7**

### CODE REVIEW

File: elevenclock/settings.py [hunk 1]
```diff
@@ -16,9 +16,9 @@
 
 try:
     import psutil
-    importedPsutil = True
+    from psutil._common import bytes2human
 except ImportError:
-    importedPsutil = False
+    print(ImportError)
```
Comment (panther7) [inline elevenclock/settings.py:21]: I am no Pythoner, i don't know that is right :)

**Comment by martinet101** [comment]
I'll check it deeper, but i think the psutil var was used somewhere else also, but i'm not sure right now

**Comment by martinet101** [comment]
I'll check it deeper, but i think the importedPsutil variable was used elsewhere

**Comment by panther7** [comment]
Added simplification for other functions (appVersion, systemInfo).

**Review by martinet101 — APPROVED** [review]

File: elevenclock/settings.py (hunk 1 above)
Comment (martinet101) [reply elevenclock/settings.py:21]: This must be some IDE-autogenerated line, it's not needed but does not harm

**Pull request merged**